src/constants/default-settings.js

```diff
@@ -22,7 +22,6 @@ import keyMirror from 'keymirror';
 
 export const ACTION_PREFIX = '@@kepler.gl/';
 export const CLOUDFRONT = 'https://d1a3f4spazzrp4.cloudfront.net/kepler.gl';
-export const STYLE_PREFIX = `${CLOUDFRONT}/map-styles`;
```

Comment:
can't you still use the STYLE_PREFIX? maybe you can set to __mapbox://styles/uberdata__